docker-compose.test.yml

```diff
@@ -0,0 +1,72 @@
+version: "3.7"
+
+x-common-variables: &test-variables
```

Comment:
Would you be ok with adding a comment with link to the docs and/or the SO post here? I love the ability to share config to avoid duplication, but it looks suspicious and strange at first.